lib/src/config/routes.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+import '../screens/categories_screen.dart';
```

Comment:
we should separate local imports with a line break

Reply:
I fixed it